siglens/siglens
# Feat: Added Download Formats Options (XML/SQL)

# Description
Added two new options (XML and SQL) for data download. These work similarly to what we already have for the formats CSV and JSON.
Also added two new tests, one for each new type of download format. These run a basic test confirming that, when we download the data, it is indeed in the format intended and with all the information needed.

Co-authored-by: Rodrigo Alves [rodrigo.lucas.alves@tecnico.ulisboa.pt]

# Testing
1. If you don't already have data loaded, go to Ingestion and Send Test Data.
2. Go to Logs, and click the download button.
3. Click Download as XML or Download as SQL.
4. This should download a file to your local computer, with the data in the format selected.

# Checklist:
Before marking your pull request as ready for review, complete the following.
